### PR TITLE
Added CodingSight, MSSQLTips, SQLShack, and RedGate Simple Talk

### PR DESCRIPTION
**Pull request by HadiFadl**
Added CodingSight, SQLShack, and MSSQLTips websites to the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Code Magic](https://blog.codemagic.io/write-for-codemagic-ci-cd/) - Applications are currently closed.
   > Technical focused articles. Specific on Flutter.
 
+- [CodingSight](https://codingsight.com) - Proposed article should be sent to [marketing@devart.com](mailto:marketing@devart.com) (payment depends on the length, research and audience)
```
Comment (malgamves) [inline README.md:46]: do you have a range for this @HadiFadl ?

**Review by malgamves — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (HadiFadl) [reply README.md:46]: @malgamves they removed the payment range from the website. It was published on [article levels](https://codingsight.com/article-levels/) and [Write for us](https://codingsight.com/write-for-us/). Unfortunately, These pages were removed recently, and I don't remember the range anymore. But, if you check the bottom of the [main page](https://codingsight.com/), you will see the "Write for US" announcement.

Comment (HadiFadl) [reply README.md:46]: @malgamves Currently, this is their only announcement: 

> CodingSight is open for new authors and partnership proposals. Send your offers and ideas to marketing@devart.com and we will get back to you at the earliest opportunity.

Comment (HadiFadl) [reply README.md:46]: @malgamves @Even for MSSQLTips, I don't know the range

**Comment by HadiFadl** [comment]
@malgamves while checking a previous conversation with CodingSight marketing team, they said that they pay from 100$ to 250$ per article depending on the article technical level.

**Comment by HadiFadl** [comment]
After checking the [following article](https://www.brentozar.com/archive/2008/12/how-start-blog/), MSSQLTips pays around 100$ per tip.

**Comment by HadiFadl** [comment]
After contacting MSSQLTips, they told me that they pay 160$ per tip.

File: README.md [hunk 2]
```diff
@@ -55,6 +55,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Code Magic](https://blog.codemagic.io/write-for-codemagic-ci-cd/) - Applications are currently closed.
   > Technical focused articles. Specific on Flutter.
 
+- [CodingSight](https://codingsight.com) - Proposed article should be sent to [marketing@devart.com](mailto:marketing@devart.com) (payment depends on the length, research and audience) - From 100$ to 250$ based on the article technical level
```
Comment (karllhughes) [inline README.md:58]: Please shorten this to:

```
- [CodingSight](https://codingsight.com) (Email [marketing@devart.com](mailto:marketing@devart.com)) - $100-$250 per piece
```

File: README.md [hunk 3]
```diff
@@ -134,6 +137,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Nanonet](https://nanonets.com/blog/write-for-us/)
   > Get paid to write about your favourite machine learning topics!
 
+- [MSSQLTips](https://www.mssqltips.com/contribute/) - 160$ per tip
```
Comment (karllhughes) [inline README.md:140]: We use the `$` up front in this repo. For consistency, please change to `$160`.

File: README.md [hunk 4]
```diff
@@ -170,6 +176,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Section.io](https://github.com/section-io/engineering-education) - $50 to $150 per article
   > Engineering education blog where Computer Science university students may contribute content for pay.
 
+- [Simple Talk](https://www.red-gate.com/simple-talk/write-for-us/) - 350$ per article
```
Comment (karllhughes) [inline README.md:179]: Should be `$350`

File: README.md [hunk 5]
```diff
@@ -181,6 +190,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Soshace](https://docs.google.com/document/d/1DZ9Hj8AcNfHI6bC4bfTDIFRNIIFnda6Mkj_n_4x3hWw/edit) - $100 per piece
   > Technical tutorials with code. Pick from a list of possible articles.
 
+- [SQLShack](https://www.sqlshack.com/about-us/) - 200$ per piece
```
Comment (karllhughes) [inline README.md:193]: Should be `$200`

**Review by karllhughes — CHANGES_REQUESTED** [review]
@HadiFadl thanks for your suggestions! I've added a few comments. Let me know when your updates are done and I'll be happy to merge this in.

**Comment by HadiFadl** [comment]
@karllhughes I made all the updates you suggested.
